src/Compilers/CSharp/Test/Semantic/Semantics/NullableReferenceTypesTests.cs

```diff
@@ -60341,13 +60372,13 @@ public void StructField_Default_NoType()
 {
     static void F()
     {
-        _ = default/*T:<null>?*/.ToString();
+        _ = default/*T:<null>!*/.ToString();
     }
 }";
             var comp = CreateCompilation(source, options: WithNonNullTypesTrue());
             comp.VerifyDiagnostics(
                 // (5,33): error CS0023: Operator '.' cannot be applied to operand of type 'default'
```

Comment:
>default [](start = 92, length = 7)

I don't expect this to change in this PR, but this also seems odd because 'default' is not a type.